code/hud/hud.h

```diff
@@ -57,7 +57,7 @@ typedef struct hud_subsys_damage
 {
 	int	str;
 	int	type;
-	char	*name;
+	char* name;
```

Comment:
I don't know if it makes sense but this could be changed to a `const char*` if possible.